docs/flags.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Tiltfile Flags and Args
+layout: docs
+---
+
+
+Tools offer flags and args to support the wide range of users needs and preferences, and dev workflows are no different. Devs on your team will want options in how they use Tilt; some are long-standing personal preferences and others will change in the middle of working on a Pull Request. With Tilt's user configuration system, you can improve a Tiltfile that encodes one standard setup to power the breadth of workflows your team needs:
+* Maintainers choose the options to expose that make sense for their project
+* Users can easily set and change options, without having to learn a Tiltfile
+* Tiltfile extensions expose the choices so they can power arbitrary customizations
```

Comment:
What are Tiltfile extensions?

Reply:
Yeah, I'm not sure what to do with this sentence; parallelism is hard. Let me think a bit.